addons/difficulties/functions/fnc_apcSkill.sqf

```diff
@@ -0,0 +1,26 @@
+#include "script_component.hpp"
+/*
+ * Author: Mike
+ * Automatically lowers APC / Tank based aiming accuracy.
+ *
+ * Call on mission start.
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [] call FUNC(apcSkill)
+ */
+
+if (!GVAR(lowerAPCSkill)) exitWith {};
+
+private _allVehicles = vehicles select {(_x isKindOf "Wheeled_APC_F" || _x isKindOf "Tank" && {!isNull gunner _x})};
```

Comment:
Hmmm, no, this will only catch vehicles that are present on settings init or when you change the setting.

Skill should be applied on each vehicle as it spawns (CBA XEH). In accordance, this function should be renamed to actually say what it does.

Reply:
1. Does it matter? We don't spawn them ever.
2. ..lowerMechanizedAccuracy?

Reply:
Did you really ask if it matters? Quality first!